src/App.vue

```diff
@@ -1,87 +1,56 @@
 <template>
 	<v-app id="inspire">
-		<v-navigation-drawer
-			v-model="drawer"
-			:clipped="$vuetify.breakpoint.lgAndUp"
-			app
-			hide-default-footer
```

Comment:
Was this removed on purpose?

Reply:
No it was not removed on purpose, I dont know how all these changes in App.vue happened I didnt modify the code except for adding some paths to my qwixx component..